ingredient_phrase_tagger/training/cli.py

```diff
@@ -14,10 +15,14 @@ def run(self):
         Generates training data in the CRF++ format for the ingredient
         tagging task
         """
-        with open(self.opts.data_path) as data_file:
+        with open(self.opts.data_path, encoding='utf-8') as data_file:
             data_reader = labelled_data.Reader(data_file)
             for row in data_reader:
-                print translator.translate_row(row).encode('utf-8')
+                # write the utf-8 encoded data directly to stdout instead of using print
```

Comment:
Clear comment, but can we capitalize the first letter and end in a period for consistency?